testutil/config.go

```diff
@@ -189,7 +195,14 @@ func MockConfig() *evergreen.Settings {
 				},
 			},
 		},
-		Credentials:        map[string]string{"k1": "v1"},
+		Credentials: map[string]string{"k1": "v1"},
+		DataPipes: evergreen.DataPipesConfig{
+			Host:         "https://url.com",
```

Comment:
I generally prefer to use "*.example.com" for fake URLs because that is reserved. Alternatively a mongodb domain would work as well. (url.com is actually owned by someone)

Reply:
Yeah these aren't actually calling anything, I just changed it to `url`.

Reply:
That works. Just don't want to set up a situation where it would be easy to accidentally call it and leak data.